src/tests/stuf/brp/test_response_mapping.py

```diff
@@ -113,16 +113,18 @@ class TestRelatedMapping(TestCase):
     def test_filter(self):
         class RelatedMappingImpl(RelatedMapping):
             entity_type = 'RELMAP'
-            mapping = {}
+            mapping = {'D': ''}
             include_related = ['A', 'B']
```

Comment:
I don't get the effect of mapping in this test? Why is the result 4, or do I miss something? What happens if mapping = {'D': 'some other value'}

Reply:
What about now? ;)